remo/dashboard/views.py

```diff
@@ -137,8 +137,10 @@ def dashboard(request):
     if user.groups.filter(name='Mentor').exists():
         args['mentees_action_items'] = ActionItem.objects.filter(user__in=my_mentees,
                                                                  resolved=False)[:10]
-        args['mentees_ng_reportees'] = User.objects.filter(
-            ng_reports__isnull=False, ng_reports__mentor=user, groups__name='Rep').distinct()
+        args['mentees_activities'] = User.objects.filter(
+            userprofile__registration_complete=True,
```

Comment:
Shouldn't we add the `ng_reports__is_null=False` here too? Otherwise it's going to display all the mentees of the user and not just the ones with reports.

Reply:
I just saw the second commit. May I ask why do we need to display all the users?

Reply:
If there is a mentee without a report, there is something wrong. So it's definitely better to list all.

Reply:
Sounds good.